### PR TITLE
make changes to README to align properly with CORDEX content. #123

### PR DESCRIPTION
**Pull request by nhsavage**
make changes to reflect that this no longer includes PRECIS data

### CODE REVIEW

**Comment by nhsavage** [comment]
feedback from Erasmo:

but I would add  in the overview something about the name PyPRECIS, e.g., it could be presented as an “homage” to the old PRECIS training on which these notebooks are based and I would also add the CSSP dataset as one of the dataset used by these notebooks which somehow support the first point (20CR is not exactly part of the PRECIS, although it could have been since it has been done with the same RCM). On your specific point, the CORDEX reference as a link to CORDEX core is very good but unfortunately it doesn’t include the CCLM simulations (the ETH ones), the first link I get from a Google search is the paper
https://gmd.copernicus.org/articles/14/5125/2021/gmd-14-5125-2021.pdf
CORDEX core has also been presented in papers, in particular there was a special issue on Clim Dyn
https://link.springer.com/journal/382/volumes-and-issues/57-5
if you also would like to add published literature, but the CORDEX CORE link should be ok for the use of REMO and RegCM3

File: README.md [hunk 1]
```diff
@@ -24,18 +24,20 @@ PyPRECIS is the python based training environment for Met Office PRECIS training
...
-Further information about PRECIS can be found on the [Met Office website](https://www.metoffice.gov.uk/precis).
+The data analysed in the notebooks is from the CORDEX-Core simulations which provide an ensemble of high-resolution (at least 25 km) regional climate change information. Further information about CORDEX-Core can be found on the [CORDEX website](https://cordex.org/experiment-guidelines/cordex-core/cordex-core-simulations/).
 
 ## Contents
 The teaching elements of PyPRECIS are contained in the `notebooks` directory. The core primary worksheets are:
 
 Worksheet | Aims
 :----: | -----------
-[1](notebooks/worksheet1.ipynb) | <li>Identify and list the names of PRECIS output data in PP format using standard Linux commands</li><li>Use basic Iris commands to load data files, and view Iris cubes</li><li>Use Iris commands to remove the model rim, select data variables and save the output as NetCDF files</li>
+[1](notebooks/worksheet1.ipynb) | <li>Identify and list the names of CORDEX output data in netCDF format using standard Linux commands.
```
Comment (hsteptoe) [inline README.md:38]: Table formatting is broken here.  I think you need to avoid new lines for text that sits inside a cell

File: README.md [hunk 2]
```diff
@@ -58,10 +60,7 @@ Three additional worksheets are available for use by workshop instructors:
 * `worksheet6example.ipynb`: Example code for Worksheet 6.
 
 ## Data
-Data relating to the PyPRECIS project is currently held internally to the Met Office.
-
-The total data volume for the core worksheets is 36.68 GB, of which ~20 GB is raw pp data. This is too large to be stored on github, or via git lfs.
-As of v2.0, the storage solution for making this data available alongside the notebooks is still under investgation.
+For information on how to access the CORDEX-Core data used in these worksheets, see: [CORDEX: How to access the data](https://cordex.org/data-access/how-to-access-the-data/). Most CORDEX data is available for unrestricted use but some is provided for non coomerical use only. Before you download any CORDEX data you must ensure you are aware of the Terms of Use for the data you are accessing.
```
Comment (hsteptoe) [inline README.md:63]: Typo - "coomerical"

**Review by hsteptoe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nhsavage) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (nhsavage) [reply README.md]: fixed

**Review by hsteptoe — APPROVED** [review]

**Pull request merged**